pdf_helper/tests/test_helper.py

```diff
@@ -17,7 +15,7 @@ def read_test_file(filename, mode="r"):
         return thefile.read()
 
 
-class TestPDFHelperUtils(TestCase):
+class TestPDFHelperUtils(BaseCommon):
```

Comment:
I know we started adopting BaseCommon everywhere but in fact it does too many things and adds quite some overhead to test cases setup even when not needed. Also, in this particular case, there was a reason to use `TestCase`. This should be restored.

For the other cases, can you please follow this approach?
https://github.com/OCA/stock-logistics-orderpoint/pull/41